scripts/check_plugin_authors.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+#
+# Script to check which plugins have a single author
+# Copyright: Red Hat 2017
+# LICENSE: GPLv3
+
+require 'json'
+require 'rest-client'
+
+def get_gem_names(search)
+  page=1
+  gems=[]
+  url="https://rubygems.org/api/v1/search.json?query=#{search}"
+
+  response = RestClient.get('https://google.com') # test connection and initialize "response"
```

Comment:
Wouldn't it be better to get rubygems.org?

Reply:
fair enough ;)